config/config.properties

```diff
@@ -218,7 +218,7 @@ metrics.enableStatistics=false
 
 # Endpoint address (hostname:port) where the ZooKeeper controlling BookKeeper for this cluster can be found at.
 # This value must be the same for all Pravega SegmentStore instances in this cluster.
-bookkeeper.zkAddress=master.mesos:2181
+bookkeeper.zkAddress=localhost:2181
```

Comment:
To be consistent with the rest of the configuration file, I also suggest you comment this line out, to indicate that that value is identical to the hardcoded default.

Reply:
That's fine with me.

Reply:
Fixed.